include/session/config/contacts.h

```diff
@@ -19,6 +20,12 @@ typedef struct contacts_contact {
     bool approved;
     bool approved_me;
     bool blocked;
+    bool hidden;
+
+    int priority;
+
+    CONVO_EXPIRATION_MODE exp_mode;
```

Comment:
I get a build error here on iOS `Must use 'enum' tag to refer to type 'CONVO_EXPIRATION_MODE` - updating it to the below works though
```
enum CONVO_EXPIRATION_MODE exp_mode;
```

Reply:
Fixed (with a `typedef` in the expiring.h header so that the `enum` isn't needed here).